### ISSUE TITLE
Sync and make [first] benefit from [second]

### ISSUE THREAD

**Issue by dbosk** [OpenSecEd/infosec#6]
If the first and second cycle courses are given in parallel, then some of the presentations the students do in the second cycle version can benefit the first cycle students.  E.g. in the second cycle version they should read and present research papers. This could benefit the first cycle students if one requirement is that the presentation should be understandable to non-experts. This is a reasonable requirement, when the students finish they should be able to consume research results and convey their importance to their colleagues --- which might have a different background.

**Comment by dbosk** [comment]
The first is now moved to [infosec-sysadm](https://github.com/OpenSecEd/infosec-sysadm/).

**Comment by dbosk** [comment]
No longer relevant.
